docs-yaml/src/yaml-support/yaml-constant.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'path';
```

Comment:
No big deal but we can change these to explicit imports if we follow the same style as the other extensions.  We can also enable TSLint to clean up some of the quotation/template string differences.

Reply:
Good idea!

Reply:
@jamarw - Updated!